src/main/java/reposense/system/LogsManager.java

```diff
@@ -21,24 +21,30 @@
     private static final int FILE_COUNT = 2;
     private static final int MEGABYTE = (1 << 20);
     private static final int MAX_FILE_SIZE_IN_BYTES = 5 * MEGABYTE; // 5MB
+    private static final ArrayList<Logger> LOGGER_LIST = new ArrayList<>();
 
     // All the log files will be store with a .log extension
     // eg. reposense.log.0, in the logs/ folder of the working directory
-    private static final String LOG_FILE_LOCATION = "./logs/";
-    private static final String LOG_FILE = LOG_FILE_LOCATION + "reposense.log";
+    private static final String LOG_FOLDER_NAME = "logs";
+    private static final String LOG_FILE_NAME = "reposense.log";
 
+    private static Path logFolderLocation;
     private static Level currentConsoleLogLevel = Level.INFO;
     private static Level currentFileLogLevel = Level.INFO;
     private static FileHandler fileHandler;
     private static ConsoleHandler consoleHandler;
 
     public static Logger getLogger(String name) {
         Logger logger = Logger.getLogger(name);
+        LOGGER_LIST.add(logger);
```

Comment:
Should the adding to LOGGER_LIST be moved down, after the checking of `logFolderLocation` (line 46)?

Reply:
No need, logger is an object. LOGGER_LIST only stores the reference of the logger.